cmake/build_mingw32.sh

```diff
@@ -8,7 +8,7 @@ else
 	CMAKE_OPTS="$CMAKE_OPTS -DLMMS_BUILD_MSYS=1"
 fi
 
-export PATH=$PATH:$MINGW/bin
+export PATH=$MINGW/bin:$PATH
```

Comment:
👍 @Umcaruje let's commit that change directly then (no PR).  That will close out your issue and then @DragonEagle can rebase his PR as needed.